# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/slrn/slrn-0.9.8.0-post.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- slrn-0.9.8.0/src/post.c	2003-07-31 19:21:18.000000000 +0200
+++ slrn/src/post.c	2003-09-04 23:56:33.000000000 +0200
@@ -1233,6 +1234,9 @@
 
    va_start(ap, fmt);
    Slrn_Post_Obj->po_vprintf(fmt, ap);
+   va_end(ap);
+
+   va_start(ap, fmt);
    vfprintf(fcc_fp, fmt, ap);
    va_end(ap);
 }
